lib/CL/pocl_llvm_build.cc

```diff
@@ -353,14 +355,84 @@ int pocl_llvm_build_program(cl_program program,
 
   POCL_MSG_PRINT_LLVM("all build options: %s\n", ss.str().c_str());
 
-  std::istream_iterator<std::string> begin(ss);
-  std::istream_iterator<std::string> end;
-  std::istream_iterator<std::string> i = begin;
-  std::vector<const char*> itemcstrs;
+  size_t replace_cnt = 0;
+  char replace_me = 0;
+
+  char *temp_options = (char *) malloc (ss.str().length() + 1);
+  strncpy (temp_options, ss.str().c_str(), ss.str().length());
+
+  /* searching for double quote in temp_options */
```

Comment:
This part still seems largely duplicated. Perhaps a function escape_quoted_whitespace() or such could be extracted?

Reply:
yes, I will do soon

Reply:
done